sdks/java/core/src/main/java/org/apache/beam/sdk/options/ValueProvider.java

```diff
@@ -91,6 +91,11 @@ public T get() {
     public boolean isAccessible() {
       return true;
     }
+
+    @Override
+    public String toString() {
+      return String.format("StaticValueProvider: %s", value);
```

Comment:
Use [`MoreObjects.toStringHelper`](https://github.com/google/guava/wiki/CommonObjectUtilitiesExplained#tostring)?

Unified format, no need to make decisions on sentences, etc. And IDE renames kind of auto-update ToStrings.

```java
return MoreObjects.toStringHelper(StaticValueProvider.class)
    .add("value", value)
    .toString();
```